languages/elixir/exercises/concept/conditionals/.docs/after.md

```diff
@@ -0,0 +1,19 @@
+You can use atoms whenever you have a set of constants you want to express. Using atoms opens a type-safe way of interacting with constant values. An atom is defined by its name, follwing the atom format:
+
+```elixir
+# All atoms are preceeded with a ':' then follow with alphanumeric snake-cased characters
+variable = :atom
+```
+
+_Atoms_ are internally represented by an integer in a lookup table, which are set automatically.  It is not possible to change this internal value.  It is generally considered to be an [anti-pattern][anti-pattern] to dynamically create atoms from user supplied input.
```

Comment:
This anti-pattern remark is a perfect example of something that you would indeed want to mention in the `after.md` document and not in the `introduction.md` document. Well done!